.storybook/public/manager.css

```diff
@@ -4,6 +4,6 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-a[id*='--variants'] {
+a[id$='--variants'] {
```

Comment:
Maybe this commit should be squashed into the previous one so we don't change this line and then change it back?